src/transformers/modeling_xlnet.py

```diff
@@ -1467,11 +1697,12 @@ def forward(
         input_mask=None,
         head_mask=None,
         inputs_embeds=None,
-        use_cache=True,
         start_positions=None,
         end_positions=None,
+        use_cache=True,
```

Comment:
Reorder for consistency

Reply:
Actually, `use_cache` should always default to `None` as in the other models. I think I missed `XLNet` in my `use_cache` refactoring :-/ : https://github.com/huggingface/transformers/pull/5194

Reply:
Can fix this in a follow-up PR though

Reply:
I'll add it to the follow-up project.

Reply:
Note: it might be always True because the config makes it default to False (which is fixable in XLNetConfig I believe).

Reply:
the problem is if someone sets the config to False => `use_cache` will still be True because the default arg of the function is wrong. 
That's why there are these lines in other models:
https://github.com/huggingface/transformers/blob/bfacb2e34f7091f948f202a3f4b638f80f280e7d/src/transformers/modeling_gpt2.py#L414

Just discussed with @TevenLeScao (think @joeddav) was also thinking about this - at the moment `XLNet` does not really cached anything and cannot speed up generation with the parameter `use_cache`: https://github.com/huggingface/transformers/pull/5632 . So at the moment it would actually be correct to just delete `use_cache` until we have implemented past key/value caching in XLNet.